### ISSUE TITLE
xfreerdp keep asking for hostname after passing remote assistance invitation file

### ISSUE THREAD

**Issue by mcajkovs** [FreeRDP/FreeRDP#5678]
I've successfully compiled commit https://github.com/FreeRDP/FreeRDP/commit/ab2b288f87635cece0c11448875a09abc772ed11 According to this tutorial https://github.com/FreeRDP/Remmina/wiki/Compile-on-Ubuntu-18.04

My goal is to use remote assistant invitation file for connection. After executing this command:
`xfreerdp invitation.msrcincident /assistance:INVITATIONCODE`

I got following error:
`
[09:54:34:315] [20357:20357] [ERROR][com.freerdp.client.x11] - error: server hostname was not specified with /v:<server>[:port]
[09:54:34:315] [20357:20357] [ERROR][com.winpr.sync.wait] - invalid hHandle.
`

Same command works for repository client (version 2.0.0-dev)

**Comment by akallabeth** [comment]
Don´t know what you did but I can connect with a build from current master ab2b288f87635cece0c11448875a09abc772ed11 without a problem.
Did you check if the filename is correct (case sensitive)?

**Comment by mcajkovs** [comment]
thanks for reply. File name is 100% correct because if I mistype it intentionally then I get different error message + whole help for xfreerdp.


```
mcajkovs@ubuntu:~/rdp_test$ ls
invitation.msrcincident


mcajkovs@ubuntu:~/rdp_test$ xfreerdp invitation.msrcincident /assistance:XXXXX /f /smart-sizing | head -5 2<&1
[13:49:38:860] [22682:22682] [ERROR][com.freerdp.client.x11] - error: server hostname was not specified with /v:<server>[:port]
[13:49:38:860] [22682:22682] [ERROR][com.winpr.sync.wait] - invalid hHandle.


mcajkovs@ubuntu:~/rdp_test$ xfreerdp NON_EXISTING_FILE /assistance:XXXXX /f /smart-sizing | head -5 2<&1
[13:50:00:114] [22684:22684] [ERROR][com.winpr.commandline] - Failed at index 1 [NON_EXISTING_FILE]: Invalid sigil

FreeRDP - A Free Remote Desktop Protocol Implementation
See www.freerdp.com for more information

Usage: xfreerdp [file] [options] [/v:<server>[:port]]
mcajkovs@ubuntu:~/rdp_test$ 
```

and if put invitation file as last argument I get different error

```
mcajkovs@ubuntu:~/rdp_test$ xfreerdp /assistance:XXXX /f /smart-sizing invitation.msrcincident | head -5 2<&1
[13:55:51:559] [22695:22695] [ERROR][com.winpr.commandline] - Failed at index 1 [/assistance:XXXX]: Invalid sigil

FreeRDP - A Free Remote Desktop Protocol Implementation
See www.freerdp.com for more information

Usage: xfreerdp [file] [options] [/v:<server>[:port]]

```

**Comment by akallabeth** [comment]
ok, got it.
we check for `msrcIncident` extension case sensitive ;)

**Comment by mcajkovs** [comment]
> ok, got it.
> we check for `msrcIncident` extension case sensitive ;)

Thanks this worked

**Comment by akallabeth** [comment]
Fixed with 8af4ddc5f2
